agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/FirstEntryPoint.java

```diff
@@ -160,6 +161,10 @@ private static String findSystemProperties() {
   }
 
   @Override
+  @SuppressFBWarnings(
+      value = "SECCRLFLOG", // CRLF injection into log messages
+      justification =
+          "Logging params cannot be controlled by an end user of the instrumented application")
```

Comment:
no idea why this warning only started showing up with this change (https://find-sec-bugs.github.io/bugs.htm#CRLF_INJECTION_LOGS)